pkg/updater/actor.go

```diff
@@ -97,7 +97,7 @@ func (act *actor[T, O, R]) ScaleInUpdate(ctx context.Context) (bool, error) {
 	}
 	obj := act.update.Del(name)
 
-	isUnavailable := !obj.IsHealthy() || !obj.IsUpToDate()
+	isUnavailable := !obj.IsReady() || !obj.IsUpToDate()
```

Comment:
Still use `IsUpToDate()`? Should not be `Synced()`?

Reply:
IsUpToDate means the instance is updated. Synced means subresources of the instance are updated